index.html

```diff
@@ -23,7 +24,15 @@
           <img src="req/mylogo.png" alt="Adil's Logo" id="myLogo" tabindex="1" aria-label="Adil's logo"/>
         </a>
         <!--Nav bar button-->
-        <img src="req/Union.png" alt="Hamburger button" id="logo"/>
+        <button class="hamburger">
+          <i class="menuIcon material-icons">menu</i>
+          <i class="closeIcon material-icons">close</i>
+        </button>
+        <ul class="menu">
+          <li><a class="menuItem" href="#">Portfolio</a></li>
+          <li><a class="menuItem" href="#">About</a></li>
+          <li><a class="menuItem" href="#">Contact</a></li>
```

Comment:
Good job with the links but let's make sure the links are take us to the related areas and when we click on them, the menu closes like mentioned on the project requirements. 👍 
<img width="817" alt="Screenshot 2023-01-31 at 3 05 22 AM" src="https://user-images.githubusercontent.com/25887253/215625237-7c0043c2-2b40-4514-9720-f7857714b437.png">